src/stylesheets/components/_accordions.scss

```diff
@@ -109,6 +152,23 @@ $accordion-border: units($theme-accordion-border-width) solid
 
 .usa-accordion__button {
   @include accordion-button-styles;
+
+  // Anchor for add/remove icons
+  position: relative;
+
+  &::after {
+    right: 0;
+    top: 0;
+    height: 100%;
+    content: "";
+    position: absolute;
```

Comment:
Is this because we're now using `place-icon()`?

If it was previously a background image and we want to keep similar functionality we should use `add-color-icon()`. Unless there's a strong case for making it a pseudoelement.

Reply:
I understand now, thanks for clarifying!